<h1>ESTIMATION AND CONFIDENCE INTERVALS</h1>

In [2]:
# data 

data=[1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]
n=15

In [3]:
import numpy as np
import scipy.stats as stats

In [8]:
mean=np.mean(data)
print('Sample Mean :',mean)

Sample Mean : 1.2386666666666666


<h5>99% Confidence Interval Using Sample Standard Deviation</h5>

In [10]:
std_sample = np.std(data,ddof=1)
print("Sample Standard Deviation :",std_sample)

Sample Standard Deviation : 0.19316412956959936


In [16]:
confidence = 0.99
t_value = stats.t.ppf((1 + confidence) / 2 , df=n-1)
print("t_value :",t_value)

t_value : 2.976842734370834


In [17]:
margin_error = t_value *(std_sample / np.sqrt(n))
print("Margin of Error : ",margin_error)

Margin of Error :  0.14846932822817596


In [18]:
lower = mean - margin_error
upper = mean + margin_error

print("99% Confidence Interval :")
print(lower , upper)

99% Confidence Interval :
1.0901973384384906 1.3871359948948425


<h4>Interpretation</h4>
<h6>We are 99% confident that the true mean durability of the print-heads lies between 1.090 million and 1.387 million characters.</h6>

<h4>Why t-Distribution is Used?</h4>
<h6>The t-distribution is used because the sample size is small (n < 30) and the population standard deviation is unknown. It provides more accurate confidence intervals for small samples.</h6>

<h5>99% Confidence Interval Using Known Population Standard Deviation</h5>

In [19]:
population_std = 0.2

In [20]:
z_value = stats.norm.ppf(0.995)
print("z-value :",z_value)

z-value : 2.5758293035489004


In [21]:
margin_error_z = z_value * (population_std / np.sqrt(n))
print("Margin of Error :",margin_error_z)

Margin of Error : 0.13301525327090588


In [22]:
lower_z = mean - margin_error_z
upper_z = mean + margin_error_z

print ("99% Confidence Interval using population standarddeviation :")
print(lower_z , upper_z)

99% Confidence Interval using population standarddeviation :
1.1056514133957607 1.3716819199375725


<h4>Interpretation</h4>
<h6>We are 99% confident that the true mean durability lies between 1.106 million and 1.372 million characters when the population standard deviation is known.</h6>

<h3>Conclusion</h3>
<h5>The confidence interval estimates the range in which the true population mean is likely to exist. The t-distribution was used when the population standard deviation was unknown, while the z-distribution was used when the population standard deviation was known. Both intervals provide an estimate of the average durability of print-heads with 99% confidence.</h5>